server/src/lib/data/update-activity.ts

```diff
@@ -6,13 +7,17 @@ export function updateActivityCompletion({
 	sql = sqlConnection,
 	input,
 }: WithSQL<{ input: ActivityUpdateInput }>) {
+	const start = input.completion_start
+		? dayjs(input.completion_start).toISOString()
```

Comment:
I wonder if `.toISOString()` works exactly as I think it does. Could be cause for issues.